operator/pkg/metrics/monitoring.go

```diff
@@ -0,0 +1,240 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"istio.io/pkg/monitoring"
+)
+
+// OperatorVersionLabel describes version of running binary
```

Comment:
nit: end comment sentences with period. https://golang.org/doc/effective_go.html#commentary

Reply:
There are many comments. I'm fixing it whenever I can. Fixed it in this file though :)